src/test/CMakeLists.txt

```diff
@@ -40,7 +38,11 @@ add_subdirectory(pybind)
 if(${WITH_RADOSGW})
   add_subdirectory(rgw)
 endif(${WITH_RADOSGW})
-add_subdirectory(rbd_mirror)
+if(WITH_RBD)
```

Comment:
maybe we should add the `if` test at where the `add_subdirectory()` was added. please note the order in which the subdirectories were added.